### PR TITLE
Dibels data model revamp

### PR DESCRIPTION
**Pull request by GabyRangelB**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

- [x] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208381159870599

### CODE REVIEW

File: src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql
Comment (cbini) [inline src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql:1]: keep these column names generic on staging: remove `mclass_` prefix. you can alias them downstream

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_pm_dashboard.sql [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            -- 'ELA Gr3',
+            -- 'ELA Gr4',
+            -- 'ELA Gr5',
+            -- 'ELA Gr6',
+            -- 'ELA Gr7',
+            -- 'ELA Gr8'
+            )
+    )
+
+select distinct
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_pm_dashboard.sql:130]: fix duplicate issue or add comment explaining why `distinct` is necessary

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql [hunk 2]
```diff
@@ -25,7 +25,18 @@ with
             e.cohort,
 
             a.admin_season as expected_test,
+            right(a.test_code, 1) as expected_round,
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:28]: move calculations below simple select targets

File: src/dbt/kipptaf/models/amplify/mclass/intermediate/int_amplify__benchmark_student_summary_unpivot.sql [hunk 3]
```diff
@@ -1,96 +1,126 @@
...
-                composite_year_growth
-            ) as 'Composite'
-        )
+            case
+                measure
+                when 'Composite'
+                then 'Composite'
+                when 'Reading Comprehension (Maze)'
+                then 'Comprehension'
+                else substr(measure, strpos(measure, '(') + 1, 3)
```
Comment (cbini) [inline src/dbt/kipptaf/models/amplify/mclass/intermediate/int_amplify__benchmark_student_summary_unpivot.sql:20]: `regexp_extract` can do this more reliably

https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#regexp_extract

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql:1]: @cbini these also exist on int_amplify__benchmark_student_summary_unpivot... fix them there too?

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql:1]: can i alias them all like this on int_amplify__all_assessments and leave the staging intact?

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/amplify/mclass/staging/stg_amplify__pm_student_summary.sql:1]: will come back later

**Review by cbini — APPROVED** [review]

**Pull request merged**